src/hooks/useOrderForm.ts

```diff
@@ -0,0 +1,132 @@
+import { useState } from 'react';
+import {
+  PHONE_REGEX,
+  MIN_QUANTITY,
+  ERROR_MESSAGES,
+} from '@/constants/validation';
+
+type FormValues = {
+  senderName: string;
+  receiverName: string;
+  receiverPhone: string;
+  quantity: number;
+  textMessage: string;
+};
+
+type FormErrors = Record<keyof FormValues, string>;
+type FormField = keyof FormValues;
```

Comment:
여기서도 작성한 FormField 타입을 활용할 수 있겠네요 😊

```suggestion
type FormErrors = Record<FormField, string>;
type FormField = keyof FormValues;
```

Reply:
제안해주신 방식을 반영해보았습니다! 감사합니다.